### PR TITLE
(Fix) #672 - remove confusing error logging in Google import

### LINKED ISSUE #672: Errors importing Google Location History
Testing on the develop branch, when importing a google takeout zip the log file is full of errors:

[INFO] Attempting to open file "file:///data/user/0/org.pathcheck.covidsafepaths/cache/Takeout-2020-04-24T11:54:32.837Z/Takeout/Location History/Semantic Location History/2020/2020_MARCH.json" again [Error: ENOENT: No content provider: /data/user/0/org.pathcheck.covidsafepaths/cache/Takeout-2020-04-24T11:54:32.837Z/Takeout/Location History/Semantic Location History/2020/2020_MARCH.json, open '/data/user/0/org.pathcheck.covidsafepaths/cache/Takeout-2020-04-24T11:54:32.837Z/Takeout/Location History/Semantic Location History/2020/2020_MARCH.json']

### PR DESCRIPTION
**Pull request by ten-sis**
What's done:
- Remove excessive logging caused confusion in ticket #672. The import is actually completed successfully
- Add an error message in case of denied access if Take out is opened from other than Google Drive folder

#### Linked issues:

Fixes #672 

#### Screenshots:

Screen Shot 2020-04-27 at 11 06 31 PM

#### How to test

To verify import is successful:
1. Upload google takeout file
2. Go to import screen
3. Click "import locations"
4. You should see the success message and no errors in the console

To check out access denied error message (Android only):
1. Download Take out file in browser from google drive mobile site
2. Go to import screen
3. Click "import locations"
4. Select Take out file from Downloads folder

### CODE REVIEW

File: app/locales/en.json [hunk 1]
```diff
@@ -21,12 +21,18 @@
...
+      "instructions_detailed": "Visit Google Takeout and export your Location History using the following settings: \n1. Delivery method: \"Add to Drive\" \n2. Frequency: \"Export once\" \n3. File type & size: \".zip\" and \"1GB\"\n4. Google sends an email when the export is ready \n5. Return here to import locations from Google Drive",
       "instructions_first": "Adding location data from Google will give you a head start on building your recent locations.",
       "instructions_second": "Before you can import, you must first \"Take out\" your location data from Google.",
       "title": "Google Maps",
-      "visit_button_text": "Visit Google Takeout"
+      "visit_button_text": "Visit Google Takeout",
+      "already_imported":"Provided Takeout file has already been imported.",
```
Comment (tstirrat) [inline app/locales/en.json:29]: Please add screenshots of these errors/status messages, so I can upload to the i18n platform, thanks!

File: app/views/Import.js [hunk 2]
```diff
@@ -41,25 +42,35 @@ const ImportScreen = props => {
...
-        if (newLocations.length) {
-          setImportResults(makeImportResults('label.import_success'));
-        } else {
-          setImportResults(makeImportResults('label.import_already_imported'));
-        }
+
+      const newLocations = await importTakeoutData(filePath);
+
+      if (newLocations.length) {
+        setImportResults(makeImportResults('import.success'));
```
Comment (tstirrat) [inline app/views/Import.js:49]: use full `t('import.success')` whenever you use i18n keys, or the i18n detection/extraction tool will not find the strings.

**Comment by sergesemashko** [comment]
Additional screenshots:
Screen Shot 2020-04-08 at 5 39 29 PM
Screen Shot 2020-04-08 at 5 37 41 PM
Screen Shot 2020-04-08 at 5 37 35 PM
Screen Shot 2020-04-08 at 5 13 34 PM
Screen Shot 2020-04-08 at 5 09 09 PM
Screen Shot 2020-04-29 at 10 20 05 PM

**Comment by sergesemashko** [comment]
@tstirrat , thanks for reviewing the PR. I incorporated the feedback, could you re-review?

File: app/views/Import.js [hunk 3]
```diff
@@ -41,25 +42,43 @@ const ImportScreen = props => {
...
+        setImportResults(
+          makeImportResults(languages.t('import.google.already_imported')),
+        );
       }
     } catch (err) {
       if (err instanceof NoRecentLocationsError) {
         setImportResults(
-          makeImportResults('label.import_no_recent_locations', true),
+          makeImportResults(
+            languages.t('import.google.no_recent_locations'),
```
[the remark this answers is not in the document]
Comment (tstirrat) [reply app/views/Import.js:59]: While you're there, setImportResults is always called with makeImportResults, perhaps they can be moved one into the other method?

**Comment by sergesemashko** [comment]
@tstirrat , good catch, thanks. The PR is updated, please re-review

**Pull request merged**